wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php

```diff
@@ -408,13 +416,18 @@ protected function parse_readme_contents( $contents ) {
 		// Use the first line of the description for the short description if not provided.
 		if ( ! $this->short_description && ! empty( $this->sections['description'] ) ) {
 			$this->short_description = array_filter( explode( "\n", $this->sections['description'] ) )[0];
+			$this->warnings['no_short_description_present'] = true;
 		}
 
 		// Sanitize and trim the short_description to match requirements.
 		$this->short_description = $this->sanitize_text( $this->short_description );
 		$this->short_description = $this->parse_markdown( $this->short_description );
 		$this->short_description = wp_strip_all_tags( $this->short_description );
-		$this->short_description = $this->trim_length( $this->short_description, 150 );
+		$short_description = $this->trim_length( $this->short_description, 150 );
+		if ( $short_description !== $this->short_description ) {
+			$this->warnings['trimmed_short_description'] = true;
```

Comment:
```suggestion
			if ( empty( $this->warnings['no_short_description_present'] ) ) {
				$this->warnings['trimmed_short_description'] = true;
			}
```

Reply:
I tested with this suggestion and it is working as expected. If there is warning `trimmed_short_description` then we can flag it for short description exceeding 150 characters.